circuit_knitting/cutting/cut_wire_to_move.py

```diff
@@ -14,11 +14,15 @@
 from __future__ import annotations
 
 from itertools import groupby
+import numpy as np
+
```

Comment:
Thanks, it was actually because of this comment that I realized we should be doing relative import of ckt anyway here.  Fixed in 0d08b59f1f66c464e066909e2f4c332871308783.